.github/workflows/test.yaml

```diff
@@ -190,20 +190,13 @@ jobs:
 
       - name: Smoke test - remote_call_rate
         run: |
-          set -x
-
-          # Start local network services
-          nix develop .#ci -c bash -c "hc-run-local-services --bootstrap-port 4422 --signal-port 4423 &"
-          # Start a TryCP instance
-          nix develop .#ci -c bash -c "source ./scripts/trycp.sh && start_trycp &"
+          # Start a sandbox conductor and run it in the background
+          nix develop .#ci -c bash -c "hc s clean && echo "1234" | hc s --piped create && echo "1234" | hc s --piped -f 8888 run &"
 
           # Run the scenario
-          RUST_LOG=warn CONDUCTOR_CONFIG="CI" MIN_PEERS=2 nix run .#remote_call_rate -- --targets targets-ci.yaml --instances-per-target 2 --duration 45 --no-progress
+          RUST_LOG=info MIN_AGENTS=2 cargo run --package remote_call_rate -- --connection-string ws://localhost:8888 --agents 2 --duration 30
```

Comment:
Should logging stay at `warn` on CI?

Reply:
Ah yes. And also `--no-progress` should be preserved.